### ISSUE TITLE
[BUG] Pre-release in main channel in apt repository

### ISSUE THREAD

**Issue by reneklacan** [Snapchat/KeyDB#446]
**Describe the bug**

KeyDB 6.3.x pre-release is in the main channel in apt repository

**To reproduce**

`apt update && apt upgrade` will upgrade KeyDB to pre-release version

**Expected behavior**

Non-stable versions should not be in the main channel of apt repository

**Additional information**

```
$ apt-cache madison keydb
     keydb | 6:6.3.1-1~bionic1 | https://download.keydb.dev/open-source-dist bionic/main amd64 Packages
     keydb | 6:6.3.0-1~bionic1 | https://download.keydb.dev/open-source-dist bionic/main amd64 Packages
     keydb | 6:6.2.2-1~bionic1 | https://download.keydb.dev/open-source-dist bionic/main amd64 Packages
     keydb | 6:6.2.1-1~bionic1 | https://download.keydb.dev/open-source-dist bionic/main amd64 Packages
     keydb | 6:6.2.0-1~bionic1 | https://download.keydb.dev/open-source-dist bionic/main amd64 Packages
     keydb | 6:6.0.18-1~bionic1 | https://download.keydb.dev/open-source-dist bionic/main amd64 Packages
     keydb | 6:6.0.16-1~bionic1 | https://download.keydb.dev/open-source-dist bionic/main amd64 Packages
     keydb | 6:6.0.13-1~bionic1 | https://download.keydb.dev/open-source-dist bionic/main amd64 Packages
     keydb | 6:6.0.9-1~bionic1 | https://download.keydb.dev/open-source-dist bionic/main amd64 Packages
     keydb | 6:6.0.8-1~bionic1 | https://download.keydb.dev/open-source-dist bionic/main amd64 Packages
     keydb | 5:5.3.3-1chl1~bionic1 | https://download.keydb.dev/open-source-dist bionic/main amd64 Packages
     keydb | 5:5.3.2-1chl1~bionic1 | https://download.keydb.dev/open-source-dist bionic/main amd64 Packages
     keydb | 5:5.3.1-1chl1~bionic1 | https://download.keydb.dev/open-source-dist bionic/main amd64 Packages
```

**Comment by benschermel** [comment]
Thanks for pointing this out, we have tagged the version for 6.3.1 here: https://github.com/Snapchat/KeyDB/releases/tag/v6.3.1 
This version contained critical fixes from 6.3.0 and has been published on our various distribution channels

**Comment by reneklacan** [comment]
@benschermel 6.3.1 crashed in our production (https://github.com/Snapchat/KeyDB/issues/445) so I think you should still be careful about that
